### PR TITLE
Updated node version and pnpm

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Updated node version and pnpm.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,10 @@
   "name": "yamada-theme",
   "version": "0.1.0",
   "private": true,
-  "packageManager": "pnpm@9.4.0+sha512.f549b8a52c9d2b8536762f99c0722205efc5af913e77835dbccc3b0b0b2ca9e7dc8022b78062c17291c48e88749c70ce88eb5a74f1fa8c4bf5e18bb46c8bd83a",
+  "engines": {
+    "node": ">=20"
+  },
+  "packageManager": "pnpm@9.6.0",
```
Comment (koralle) [inline package.json:8]: @hirotomoyamada 

> The hash is optional but strongly recommended as a security practice. 

See: https://github.com/nodejs/corepack/blob/main/README.md#when-authoring-packages

**Review by koralle — DISMISSED** [review]

**Review by 108yen — CHANGES_REQUESTED** [review]
Remove the `version` listed here. It conflicts with the `packageManeger` version.
https://github.com/yamada-ui/yamada-theme/blob/941f76192633eade405d01afc8e2516a25ff7295/.github/composite-actions/install/action.yml#L9-L10

File: package.json (hunk 1 above)
Comment (hirotomoyamada) [reply package.json]: Fixed.

**Comment by hirotomoyamada** [comment]
@108yen 

Fixed.

**Review by 108yen — APPROVED** [review]

**Pull request merged**